### DISCUSSION TITLE
关于o1模型的响应时间

### DISCUSSION THREAD

**Discussion by maosasagawa** [lobehub/lobehub#4103] [General Question] [answered]
对于一些复杂问题o1模型可能响应时间较长，在等到回复之前lobechat就报错超时，提示未等到上游服务器回应，但上游服务器的确回应了不过时间较长。是否可以考虑延长o1模型的等待时间？

**Comment by arvinxx** [comment, marked as answer]
这个跟LobeChat无关，代码里没有设定超时时间的。

如果你是用 vercel 部署的话，Vercel 有最长 X s 的响应时长限制，无法突破（除非你额外花钱升级plan）。

解决方案是你切换成从浏览器发起请求
